go.mod

```diff
@@ -34,21 +34,23 @@ require (
 	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 	gopkg.in/natefinch/npipe.v2 v2.0.0-20160621034901-c1b8fa8bdcce
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.23.4
-	k8s.io/apimachinery v0.23.4
+	k8s.io/api v0.23.5
+	k8s.io/apimachinery v0.23.5
 	k8s.io/client-go v1.5.2
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
-	knative.dev/client v0.28.0
-	knative.dev/eventing v0.28.4
+	knative.dev/client v0.29.0
+	knative.dev/eventing v0.29.2-0.20220420140829-ce4fe0990d23
 	knative.dev/hack v0.0.0-20220128200847-51a42b2eb63e
-	knative.dev/pkg v0.0.0-20220222214539-0b8a9403de7e
-	knative.dev/serving v0.28.4
+	knative.dev/networking v0.0.0-20220120045035-ec849677a316 // indirect
+	knative.dev/pkg v0.0.0-20220222214439-083dd97300e1
+	knative.dev/serving v0.29.5
 )
 
 replace (
+	// Tekton Triggers imports old google/cel-go, should be fixed with tektoncd/cli >=v0.24.x transitively
+	github.com/google/cel-go => github.com/google/cel-go v0.11.2
```

Comment:
The older `cel-go` is transitively used by `tektoncd/triggers -> tentocd/cli` that version contains an older import path for `antlr4` lib. That is clashing with `cloudevents/sdk-go/v2/sql` import. Triggers have the newer version used on current `main` branch, therefore I'd assume it's a matter of few days (maybe weeks :)) to get the issue resolved through newer versions.

https://github.com/cloudevents/sdk-go/blob/release-v2.8/sql/v2/go.mod#L6